serverless/Dockerfile.j2

```diff
@@ -0,0 +1,21 @@
+FROM node:{{NODE_MAJOR}}-alpine
+
+ARG PIPENV_VERSION
+ARG PIP_VERSION
+
+RUN echo "Starting ..." && \
+    echo "Install system dependencies for python and pip" && \
+    apk add --update -q --no-cache bash build-base curl git libffi-dev make openldap-dev openssh-client python3 py3-pip rsync tzdata zip && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
+    pip3 install pip==${PIP_VERSION} && \
+    echo "Done base install!" && \
+    echo "Install basics Python tools" && \
+    pip install --upgrade pipenv==${PIPENV_VERSION} && \
+    echo "Done basic python tools install" && \
+    echo "Install serverless" && \
+    npm install -g serverless@${SERVERLESS_VERSION} && \
```

Comment:
Shouldn't there be `ARG SERVERLESS_VERSION` with the 2 other `ARG`s?

Reply:
Indeed, it's been added.